src/components/products/review/CreateReview.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import submitProductReview from '../../../redux/actions/ratingAndFeebackActions';
+import StarRating from './StarRatings';
+
+const CreateReview = ({ productId }) => {
+  const [rating, setRating] = useState();
+  const [feedback, setFeedback] = useState('');
+  const [isSubmitted, setIsSubmitted] = useState(false);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const storedStatus = localStorage.getItem('reviewSubmitted');
+    if (storedStatus === 'true') {
+      setIsSubmitted(true);
+    }
+    return () => {
+      localStorage.removeItem('reviewSubmitted');
+    };
+  }, []);
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    dispatch(submitProductReview({ productId, rating, feedback }));
+    if (rating !== 0 && feedback !== '' && feedback.length > 4) {
```

Comment:
What will happen if this line `if (rating !== 0 && feedback !== '' && feedback.length > 4) ` become **False** ❓ 

coz I can see you are doing **dispatch** before the check.